rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/configurations/UnityAttachToEditorFactory.kt

```diff
@@ -12,6 +13,7 @@ open class UnityAttachToEditorFactory(type: ConfigurationType)
     override fun isConfigurationSingletonByDefault() = true
     override fun getName() = "Debug Unity Editor"
     override fun getIcon() = UnityIcons.RunConfigurations.AttachAndDebug
+    override fun isApplicable(project: Project) = project.isUnityProject()
```

Comment:
Consider using com.jetbrains.rider.UnityProjectDiscovererKt#isUnityClassLibraryProject
One may want to attach from UnityClassLib project.

Reply:
Check if you can attach from our rider-unity solution.

Reply:
Good call. I looked if `isUnityProject` worked with generated and non-generated projects, but missed projects outside of the Unity solution structure. All this does is move the configurations into the "Other" section of the new drop down for projects where it's not appropriate, so it doesn't affect much, and you can still use them and connect from any project type. I'll add the check.

<img width="361" alt="Screenshot 2020-07-02 at 2 09 39 pm" src="https://user-images.githubusercontent.com/222659/86363138-52374600-bc6e-11ea-9750-1943c5e853f5.png">